src/astro/databases/postgres.py

```diff
@@ -70,20 +73,31 @@ def load_pandas_dataframe_to_table(
         :param if_exists: Strategy to be used in case the target table already exists.
         :param chunk_size: Specify the number of rows in each batch to be written at a time.
         """
-        schema = None
         if target_table.metadata and target_table.metadata.schema:
             self.create_schema_if_needed(target_table.metadata.schema)
-            schema = target_table.metadata.schema.lower()
-        source_dataframe.to_sql(
-            target_table.name,
-            schema=schema,  # type: ignore
-            con=self.sqlalchemy_engine,
+
+        # create an empty table
```

Comment:
@dimberman - We have a standard function to create tables - https://github.com/astronomer/astro-sdk/pull/538. I think it would be a good idea to standardize the table creation.

Reply:
@tatiana this should also address your question about offering support for custom columns as well